plugins/examples/rest-handler/build.gradle

```diff
@@ -43,6 +59,7 @@ integTest {
   dependsOn exampleFixture
   runner {
     nonInputProperties.systemProperty 'external.address', "${-> exampleFixture.addressAndPort}"
+    nonInputProperties.systemProperty('tests.rest.spec_root', project.sourceSets.test.output.resourcesDir)
```

Comment:
As mentioned in my other comment, I don't see the need for the addition of this system property everywhere. Do we ever set this to any other location?

Reply:
it is see comment above.